kuma/settings/common.py

```diff
@@ -932,6 +938,13 @@ def pipeline_one_scss(slug, **kwargs):
             'async': True,
         },
     },
+    'codepen-iex': {
+        'source_filenames': (
+            'js/components/codepen-iex/lib/ei.js',
```

Comment:
I don't feel strongly about whether we host this or pull it from a CDN. But we should ensure that it isn't part of the first load. I'm not sure how our jinja2 macros for scripts work, but hopefully we can use a script defer and/or script async for this and just make sure that the "Try it live" button is disabled by default until the script loads.